code/src/GridAPI/Rows.ts

```diff
@@ -111,4 +111,19 @@ namespace GridAPI.Rows {
 
         return output;
     }
+
+    /**
+     * Function that will set start index of row.
+     *
+     * @export
+     * @param {string} gridID ID of the Grid where the change will occur.
+     * @param {number} startIndex New row start index.
+     */
+    export function UpdateRowNumber(gridID: string, startIndex: number): void {
```

Comment:
UpdateRowNumber name might be confusing as it sounds like you will be able to update A row number instead of changing the startIndex. (Not sure if I'm confused with this or this is confusing me 😅)

Reply:
Could be UpdateStartIndex?